tests/e2e/kubetest2-kops/deployer/up.go

```diff
@@ -130,6 +130,7 @@ func (d *deployer) createCluster(zones []string, adminAccess string, yes bool) e
 		"--kubernetes-version", d.KubernetesVersion,
 		"--ssh-public-key", d.SSHPublicKeyPath,
 		"--set", "cluster.spec.nodePortAccess=0.0.0.0/0",
+		"--set", "spec.nodeProblemDetector.enabled=true",
```

Comment:
@upodroid Could we add a `Feature` tag on this, seems like a niche thing to test all the time.